src/components/Onboarding/content.js

```diff
@@ -195,7 +198,32 @@ highlights.main = [
       small: mainnetSmall2,
       large: mainnetLarge2,
       color: '#F9FAFC',
-      buttonColor: '#FF9780',
+      buttonColor: '#8E54A5',
+    },
+  },
+  {
+    title: {
+      small: null,
+      large: 'Buying HNY ⚖️',
```

Comment:
should we use a sun instead of that other character?

Reply:
Ah yeah good catch, maybe the 🍯  might make sense?

Reply:
yeah probably!